packages/lib/test/src/artifacts/ZWeb3.test.js

```diff
@@ -114,34 +114,34 @@ describe('ZWeb3', function() {
           }).timeout(5000);
         });
       });
+    });
 
-      describe('checksum address', function() {
-        context('when address has uppercase hexa letters', function() {
-          context('when address is not well checksummed', function() {
-            it('fails', function() {
-              const address = '0x82154A1cbcAb0461D4C12ee9Cb890244eead6d9F';
-              () =>
-                ZWeb3.toChecksumAddress(address).should.throw(
-                  'Error',
-                  /is not a valid Ethereum address or it has not been checksummed correctly/,
-                );
-            });
-          });
-
-          context('when address has been checksummed properly', function() {
-            it('returns checksummed address', function() {
-              ZWeb3.toChecksumAddress(accounts[0]).should.be.eq(accounts[0]);
-            });
+    describe('checksum address', function() {
```

Comment:
This is unrelated to the PR purpose but I was editing this file and noticed that this describe block was incorrectly placed within the previous describe block.